src/views/schemaBrowser/contributes.json

```diff
@@ -95,6 +95,11 @@
         "title": "Set filter",
         "category": "Db2 for i",
         "icon": "$(search)"
+      },
+      {
+        "command": "vscode-db2i.viewAuthorities",
+        "title": "View authorities",
```

Comment:
Seems like "View authorities" should be: "View Permissions"

Reply:
I changed "View authorities" to "View permissions"